Examples/Examples/BasemapGalleryExampleView.swift

```diff
@@ -52,6 +75,17 @@ struct BasemapGalleryExampleView: View {
             }
     }
     
+    /// A button that allows a user to close a sheet.
+    ///
+    /// This is especially useful for when the sheet is open an iPhone in landscape.
+    private var xButton: some View {
+        Button {
+            showBasemapGallery.toggle()
+        } label: {
+            Image(systemName: "xmark")
```

Comment:
In the other sheets in the toolkit a "Done" button is used (in the Popup) and in the FloorFilter, a `xmark.circle` image is used.  The HIG states: _Position Done and Cancel buttons as people expect. Typically, a Done or Dismiss button belongs in a sheet’s top-right corner (in a left-to-right layout) or top-left corner (in a right-to-left layout)._

```suggestion
            Text("Done")
```